app/models.py

```diff
@@ -0,0 +1,17 @@
+from sqlalchemy import Column, Integer, String, create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+DATABASE_URL = "postgresql://shopadmin:testpassword@localhost:5432/shop"
+
+engine = create_engine(DATABASE_URL)
+SessionLocal = sessionmaker(autocommit=False, autoflush=False, bind=engine)
```

Comment:
For future code, you'd probably want to have async DB-sessions. Otherwise, you could stall your async functions.

Reference: https://docs.sqlalchemy.org/en/21/orm/extensions/asyncio.html#sqlalchemy.ext.asyncio.async_sessionmaker